plugins/woocommerce/src/Admin/Features/ShippingPartnerSuggestions/DefaultShippingPartners.php

```diff
@@ -285,6 +285,15 @@ public static function get_all() {
 				'learn_more_link'   => 'https://woocommerce.com/products/shipping/',
 				'is_visible'        => array(
 					self::get_rules_for_countries( array( 'US' ) ),
+					(object) array(
+						'type' => 'not',
+						'operand' => array(
+							(object) array(
+								'type'    => 'plugins_activated',
+								'plugins' => array( 'woocommerce-shipping' ),
+							),
+						),
+					),
```

Comment:
Controls what shipping extensions are recommended in the "Select your shipping options" WC Home task:

Before (suggesting WCS&T):

<img width="691" alt="Screenshot 2024-06-07 at 00 28 46" src="https://github.com/woocommerce/woocommerce/assets/1759681/fc7f4c8b-f98f-4600-a39a-ebb5804ddbb0">

After (no WCS&T suggestion):

<img width="696" alt="Screenshot 2024-06-07 at 00 29 04" src="https://github.com/woocommerce/woocommerce/assets/1759681/8b80cd6b-0ebf-4119-b69b-a0630a72a5f2">